Energy Efficiency EACs Using Pre and Post Measurements.md

```diff
@@ -0,0 +1,516 @@
+   </td>
+  </tr>
+  <tr>
+   <td>1000
+   </td>
+   <td>                       64 
+   </td>
+  </tr>
+  <tr>
+   <td>2500
+   </td>
+   <td>                       66 
+   </td>
+  </tr>
+  <tr>
+   <td>5000
+   </td>
+   <td>                       67 
+   </td>
+  </tr>
+  <tr>
+   <td>≥7500
+   </td>
+   <td>                       68 
+   </td>
+  </tr>
+</table>
+
+
+Where baseline data logging is conducted, the same circuits will be monitored post retrofit for side-by-side comparison.  In such cases, the difference between measured post energy consumption to baseline energy consumption will be used to verify the true reduction;
+
+
+    _Saved Annual Electricity (kWh/yr) = Measured Baseline Annual Electricity – Measured Proposed Annual Electricity_
+
+Where 
+
+
+    _Measured Baseline Annual Electricity = ∑ Measured Baseline Circuit kW x Measurement Interval (minutes) / 60 minutes per hour **and**_
+
+
+    _Measured Proposed Annual Electricity = ∑ Measured Proposed Circuit kW x Measurement Interval (minutes) / 60 minutes per hour <span style="text-decoration:underline;">if power</span>_ is measured
+
+_Or_
+
+
+    _Measured Baseline Annual Electricity = ∑ Measured Baseline Circuit Amperage x Baseline Circuit Voltage / 1000 W per kW x Measurement Interval (minutes) / 60 minutes per hour **and**_
+
+
+    _Measured Proposed Annual Electricity = ∑ Measured Baseline Circuit Amperage x Baseline Circuit Voltage / 1000 W per kW x Measurement Interval (minutes) / 60 minutes per hour <span style="text-decoration:underline;">if current</span>_ is measured
+
+Measurements will be taken for a full year during the Proposed period, and optimally for a full year in the Baseline period.  However, if the baseline monitoring period is not one full year, the Baseline Period Electricity will be extrapolated to annual consumption;
+
+
+    _Measured Baseline Annual Electricity = Measured Baseline Period Electricity / Days in Monitoring Period x 365 days_
+
+Where _Measured Baseline Period Electricity_ is calculated in the same manner as Annual Electricity noted above.
+
+The total of the _Saved Annual Electricity_ above will be the final annual savings where 100% of the lighting intervention is metered.  If a representative sampling has been done, the savings will be extrapolated using the Calculated savings from the line-by-line audit used for the Energy Savings Calculation;
+
+
+    _ Total Saved Annual Electricity = Sample Saved Annual Electricity / Sample Calculated Annual Electricity Savings * Total Calculated Saved Annual Electricity_
+
+If only post retrofit data logging is available, the data will be validated against the Proposed Annual Electricity, and compared to the calculated Baseline Annual Electricity_ _for the same fixtures monitored to verify savings;
+
+
+    _Saved Annual Electricity = Baseline Calculated Fixture Demand (kW) * Annual Hours of Operation (hrs/yr) - Measured Proposed Annual Electricity (kWh/yr)_
```

Comment:
How long will we assume that this project generates EACs? As building codes improve, at some point I imagine the counterfactual would be also LED lights. Should there be a limit on the "EAC mintable" lifetime of this project to take that into account? Or is there another way to account for the changing counterfactual of the baseline?